src/src/numbers/Main.java

```diff
@@ -0,0 +1,20 @@
+package src.numbers;
+
+import java.util.Scanner;
+
+public class Main{
+
+	public static void main(String[] args) {
+		final Scanner scanner = new Scanner(System.in);
+		Request request = new Request(Request.INSTRUCTIONS_PRINT_REQUEST);
+		Response response = new Response(request);
```

Comment:
Why you created `Request` and `Response` objects?

Reply:
Because first console output must contain welcome message and instructions, that are stored in "INSTRUCTIONS" field in "Response" class. Access modifier of "INSTRUCTIONS" field is private and the only way to print instructions, without users request, is to create request for instructions printing manualy to get required response, which must be created manualy too.